src/sentry/static/sentry/app/views/eventsV2/landing.tsx

```diff
@@ -0,0 +1,197 @@
+import DocumentTitle from 'react-document-title';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import * as ReactRouter from 'react-router';
+import {Params} from 'react-router/lib/Router';
+import {Location} from 'history';
+
+import {t} from 'app/locale';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import SentryTypes from 'app/sentryTypes';
+import {Organization} from 'app/types';
+import localStorage from 'app/utils/localStorage';
+import withOrganization from 'app/utils/withOrganization';
+
+import GlobalSelectionHeader from 'app/components/organizations/globalSelectionHeader';
+import Banner from 'app/components/banner';
+import Button from 'app/components/button';
+import Feature from 'app/components/acl/feature';
+import SearchBar from 'app/views/events/searchBar';
+import NoProjectMessage from 'app/components/noProjectMessage';
+
+import {PageContent, PageHeader} from 'app/styles/organization';
+import space from 'app/styles/space';
+
+import Events from './events';
+import SavedQueryButtonGroup from './savedQuery';
+import EventView from './eventView';
+import EventInputName from './eventInputName';
+import {DEFAULT_EVENT_VIEW_V1} from './data';
+import QueryList from './queryList';
+import DiscoverBreadcrumb from './breadcrumb';
+
+const DISPLAY_SEARCH_BAR_FLAG = false;
+
+type Props = {
+  organization: Organization;
+  location: Location;
+  router: ReactRouter.InjectedRouter;
+  params: Params;
+};
+
+class DiscoverLanding extends React.Component<Props> {
+  static propTypes: any = {
+    organization: SentryTypes.Organization.isRequired,
+    location: PropTypes.object.isRequired,
+    router: PropTypes.object.isRequired,
+  };
+
+  state = {
+    isBannerHidden: localStorage.getItem('discover-banner-dismissed'),
+  };
+
+  getDocumentTitle = (name: string | undefined): Array<string> => {
+    return typeof name === 'string' && String(name).trim().length > 0
+      ? [String(name).trim(), t('Discover')]
+      : [t('Discover')];
+  };
+
+  handleClick = () => {
+    localStorage.setItem('discover-banner-dismissed', true);
+    this.setState({isBannerHidden: true});
+  };
+
+  renderBanner() {
+    const bannerDismissed = localStorage.getItem('discover-banner-dismissed');
+
+    if (bannerDismissed) {
+      return null;
+    }
+
+    const eventView = EventView.fromEventViewv1(DEFAULT_EVENT_VIEW_V1);
+
+    const to = {
+      pathname: location.pathname,
+      query: {
+        ...eventView.generateQueryStringObject(),
+      },
+    };
+
+    return (
+      <Banner
+        title={t('Discover')}
+        subtitle={t('Customize your query searches')}
+        onCloseClick={this.handleClick}
+      >
+        <Button
+          to={to}
+          onClick={() => {
+            trackAnalyticsEvent({
+              eventKey: 'discover_v2.prebuilt_query_click',
+              eventName: 'Discoverv2: Click a pre-built query',
+              organization_id: this.props.organization.id,
```

Comment:
Does this need to be cast to an int?

Reply:
Going to resolve this in a follow up PR for all usage of `trackAnalyticsEvent()` in Discover 2